packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/setBilling.ts

```diff
@@ -14,13 +18,21 @@ type ContractInput = [
   },
 ]
 
-const makeContractInput = async (input: StarknetSetBillingInput): Promise<ContractInput> => {
+const makeContractInput = async (
+  input: StarknetSetBillingInput,
+  ctx: ExecutionContext,
+): Promise<ContractInput> => {
+  if (ctx.rdd) {
+    const contract = ctx.rdd[CONTRACT_TYPES.AGGREGATOR][ctx.contractAddress]
+    input = contract.billing
```

Comment:
@archseer current RDD doesn't contain those two additional fields. Should RDD tool be adjusted accordingly?

Reply:
Similar to Terra we should yeah